.travis.yml

```diff
@@ -0,0 +1,11 @@
+language: minimal
+
+git:
+  depth: 1
```

Comment:
Did we say it might be better to change this to 2/3?

Reply:
Yes - https://travis-ci.com/getyoti/yoti-joomla/builds/117886665 failed due to commits in quick succession - changed to 3 in ee1b1a1